src/commands/login.ts

```diff
@@ -23,6 +25,8 @@ export default class Login extends Command {
       password,
     })
 
+    const path = `${os.homedir()}/.config/artsy`
+    fs.writeFileSync(path, result.access_token)
```

Comment:
Perhaps consider `~/.netrc` ? There’s probably libs for that too.

Reply:
(Also can’t assume `.config` exists.)

Reply:
Re: .netrc, we have a ticket for that: https://github.com/artsy/artsy-cli/projects/1#card-24981275. I think stuffing it into the file system is an iteration toward that goal.

Reply:
Yeah, this PR was intentionally naive, but then I got spec failures and now I'm sorta changing my mind. I've got some work locally where I used testdouble.js to mock out `fs` and `os`, but then I run specs and everything blows up, haha. I should have seen this coming, but when you mock those out, they are mocked for all the code, not just the SUT. I learned this lesson before with pear but then forgot.

https://github.com/jonallured/pear/blob/master/test/pear-data.test.ts

With pear I actually extracted some units that wrap `fs` and `os` and are under my control that I could then safely mock. Going to take a step back and work on that so that how we persist does not matter to this test. We could use ~/.netrc or a config file or whatever.

Reply:
With something like https://www.npmjs.com/package/netrc you only need to mock that API, FS tests are already done by the lib.